batcher/aligned-batcher/src/types/batch_queue.rs

```diff
@@ -87,9 +110,428 @@ impl Ord for BatchQueueEntryPriority {
         if ord == std::cmp::Ordering::Equal {
             self.nonce.cmp(&other.nonce).reverse()
         } else {
-            ord
+            ord.reverse()
```

Comment:
Cmp function can be changed instead of using a reverse here

Reply:
solved in 75cdc89fd644906855f13a890c4867095acfc5b6